controllers/restic.go

```diff
@@ -134,125 +139,97 @@ func (r *VeleroReconciler) buildResticDaemonset(velero *oadpv1alpha1.Velero, ds
 	if ds == nil {
 		return nil, fmt.Errorf("ds cannot be nil")
 	}
-	ds.Spec = appsv1.DaemonSetSpec{
-		Selector: ds.Spec.Selector,
-		UpdateStrategy: appsv1.DaemonSetUpdateStrategy{
-			Type: appsv1.RollingUpdateDaemonSetStrategyType,
-		},
-		Template: v1.PodTemplateSpec{
-			ObjectMeta: metav1.ObjectMeta{
-				Labels: map[string]string{
-					"component": Restic,
-				},
+
+	resticDaemonSetName := ds.Name
```

Comment:
Preserve namespace as well?

Reply:
ns pis passed as an arg to the install pkg command